### PR TITLE
Adjusting docs for 1.5.1

### PR DESCRIPTION
**Pull request by cromefire**
- [x] Changelog
- [x] README
- [x] doc site

### CODE REVIEW

**Comment by cromefire** [comment]
@prateekbh Can you have a look at the new components and update the `README` for them?

**Comment by cromefire** [comment]
@prateekbh It might be a stupid idea but I was adding font ligatures to my IDE and thought: Woudn't this be great for the docs?
(Font that looks cool: [Fira Code](https://github.com/tonsky/FiraCode#solution))

**Comment by prateekbh** [comment]
but a google font would vbe certainly easier to deal with

**Comment by cromefire** [comment]
Don't know if google has a font for that, I will look if it makes a difference, but that's just a thought and subject of a different PR.

So every thing else is okay? I'll do the documentation page and you should update the component list and then we should be good.

**Comment by prateekbh** [comment]
actually, i want to remove the component list from readme 😆, since we have a full fledged site now

**Comment by cromefire** [comment]
Also reasonable

**Comment by cromefire** [comment]
The site should maybe linked from the readme

**Comment by cromefire** [comment]
By looking at the docs, is the typography broken?

**Comment by cromefire** [comment]
I'm doing the docs right now and have the problem that `Typography` uses `span` instead of `div`. The docs do not specify the tag, there should be something like a `div` prop. I think I'll do a PR.

**Comment by prateekbh** [comment]
FWIW there should be a tag prop, where i can give the tag name. `Button` component does something like this

**Comment by cromefire** [comment]
Yeah

**Comment by artipost** [comment]
Failed pictures (27): [failed-pictures.zip](https://gist.githubusercontent.com/artipost/bfb4ba41ad5a663463e563955bb47fd5/raw/b41dfc38298d3308379ad62591183158171969f4/failed-pictures.zip)

##
:postbox: Posted by [artipost.io](https://artipost.io), through Travis build [414864335](https://travis-ci.org/prateekbh/preact-material-components/builds/414864335), for commit [4f0e62e](https://github.com/prateekbh/preact-material-components/pull/924/commits/4f0e62e314e2524f16b3d0d621a360bd482b3daf).

File: docs/src/components/app/style.scss [hunk 1]
```diff
@@ -68,14 +68,46 @@
...
+  }
 }
 
 :global .mdc-typography--body {
   line-height: 28px;
 }
+
+@font-face {
+  font-family: 'Fira Code';
+  src: url('https://cdn.rawgit.com/tonsky/FiraCode/1.205/distr/eot/FiraCode-Regular.eot');
```
Comment (prateekbh) [inline docs/src/components/app/style.scss]: please ad a preload to these chrome's font in template.html

File: docs/src/routes/tabbar/index.js [hunk 2]
```diff
@@ -48,11 +48,13 @@ export default class TabsPage extends Component {
...
         <div className="mdc-typography--display1">Sample code </div>
-        <CodeBlock>{sample}</CodeBlock>
+        <CodeBlock>
+          <code class="lang-js">{sample}</code>
+        </CodeBlock>
 
         <div className="mdc-typography--display1">Original documentation</div>
         <div className="mdc-typography--body">
-          This component encapsulates
+          This component encapsulates{' '}
```
Comment (prateekbh) [inline docs/src/routes/tabbar/index.js]: ? for space? put it before mdc-tab-bar

Comment (cromefire) [reply docs/src/routes/tabbar/index.js]: Prettier did this automatically

File: docs/src/components/app/style.scss (hunk 1 above)
Comment (cromefire) [reply docs/src/components/app/style.scss]: As it's not necessary and work well without it I don't want it to be preloaded, but loaded after everything else, that's by design

File: docs/src/components/app/style.scss [hunk 3]
```diff
@@ -68,14 +68,46 @@
     padding: 2px;
     margin: 2px;
   }
+  .code-snippet {
+    font-variant: common-ligatures;
```
Comment (prateekbh) [inline docs/src/components/app/style.scss]: Font display : swap?

Comment (cromefire) [reply docs/src/components/app/style.scss]: It has to go into the `@font-face`, but I've inserted it.

**Comment by artipost** [comment]
Failed pictures (27): [failed-pictures.zip](https://gist.githubusercontent.com/artipost/3121f828f44e71516515324cd2f9d6de/raw/cba57f0e9080955b4894be910a5ef644efcdada5/failed-pictures.zip)

##
:postbox: Posted by [artipost.io](https://artipost.io), through Travis build [414914615](https://travis-ci.org/prateekbh/preact-material-components/builds/414914615), for commit [6b5b296](https://github.com/prateekbh/preact-material-components/pull/924/commits/6b5b296e129e5dcc12c3fbf1f3506e712526b8a0).

**Comment by cromefire** [comment]
Fixed the `README` please control the docs so that we can merge

**Comment by prateekbh** [comment]
@cromefire I guess everything is passing, is this good to go from your side?

**Comment by cromefire** [comment]
Checked everything again seems okay

**Comment by cromefire** [comment]
I think we can risk a stable release

**Pull request merged**